msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -1,170 +1,180 @@
+|**Chats** | Group, 1:1 | New and existing | Meeting chats not supported; external members supported |
+
+## Content scope for import
+
+The following table provides the content scope.
+
+|In-scope | Out-of-scope|
+|----------|--------------------------|
+|Team and channel messages|At mentions|
+|Created time of the original message|Announcements|
+|Inline images as part of the message|Videos|
+|Links to existing files in SPO or OneDrive|Code snippets|
+|Messages with rich text|Sticker|
+|Message reply chain|Quotes|
+|High throughput processing|Cross posts between channels|
+|1:1 and group chat messages||
+|Shared and private channels||
+|Reactions and emojis||
 
 ## Prerequisites
 
-### Analyze and prepare message data
+Before you set up your Microsoft 365 tenant:
 
-* Review the third-party data to decide what is migrated.  
-* Extract the selected data from the third-party chat system.  
-* Map the third-party chat structure to the Teams structure.  
-* Convert import data into format needed for migration.  
+* Verify that a Microsoft 365 tenant exists for the import data. For more information on setting up a Microsoft 365 tenancy for Teams, see [prepare your Microsoft 365 tenant](../../concepts/build-and-test/prepare-your-o365-tenant.md).
+* Verify that team members are in Microsoft Entra ID. For more information, see [add a new user](/azure/active-directory/fundamentals/add-users-azure-active-directory) to Microsoft Entra ID.
 
-### Set up your Microsoft 365 tenant
+## Import historical messages into Teams
 
-* Ensure that a Microsoft 365 tenant exists for the import data. For more information on setting up a Microsoft 365 tenancy for Teams, see [prepare your Microsoft 365 tenant](../../concepts/build-and-test/prepare-your-o365-tenant.md).
-* Make sure that team members are in Microsoft Entra ID. For more information, see [add a new user](/azure/active-directory/fundamentals/add-users-azure-active-directory) to Microsoft Entra ID.
+You can import historical messages seamlessly, in both the existing and newly created channels or chats by performing the following steps:
 
-## Step 1: Create a team
+1. [Create a new channel or chat or use an existing one](#step-1-create-a-new-channel-or-chat-or-use-an-existing-one)
+1. [Enable migration mode to import messages](#step-2-enable-migration-mode-to-import-messages)
+1. [Check migration status](#step-3-check-migration-status)
+1. [Import messages](#step-4-import-messages)
+1. [Complete migration](#step-5-complete-migration)
+1. [Verify migration mode completion](#step-6-verify-migration-mode-completion)
 
-Since you're migrating existing data, maintaining the original message timestamps, and preventing messaging activity during the migration process are key to recreating the user's existing message flow in Teams. This is achieved as follows:
+### Step 1: Create a new channel or chat or use an existing one
```

Comment:
Seems like the document now only demonstrates new Import V2 APIs, with no reference to the existing Import V1 path for team/channels where team and channels are created in migration mode.
Can we have that referenced in this document as well?

Import V1 will still be available with new Import V2 roll out.

The path would be 
1. Create Team/StandardChannel in Migration Mode
or Start Migration mode on Existing Chat/Channel or supported types
2. Import Messages
3. Complete Migration

Reply:
1. start migration 
v1. create new team & standard channel in migration mode
v2. start migration mode on existing chat or channel

2. Import Message

3. CompleteMigration
v1. complete migration for standard channel, general channel then team -> in respective order
then add members
v2. complete migration on chat or channel, no operations is blocked if chat/channel starts migration through v2 apis.